sdk/keyvault/keyvault-certificates/tests.yml

```diff
@@ -13,6 +13,29 @@ jobs:
     parameters:
       PackageName: "@azure/keyvault-certificates"
       ResourceServiceDirectory: keyvault
+      Matrix:
```

Comment:
Is there any guidance for us to avoid having these configs in multiple places long-term? This matrix is the same one that is in the integration-test config in the `eng` folder, and I'll be adding similar configuration to even more packages... I think we should avoid repeating ourselves in many places. It'd be great if we could stash this config somewhere in the repo once and just reference it, but for now this will work.

CC @KarishmaGhiya is there a preferred solution?

Reply:
The preferred solution is to move them to the main template, but let's do that later.